library/Pipeline/src/Juvix/Pipeline/ToHR/Sig.hs

```diff
@@ -1,35 +1,29 @@
-module Juvix.ToCore.FromFrontend.Transform.Sig (transformSig) where
+{-# LANGUAGE LiberalTypeSynonyms #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
can these be moved to the `package.yaml`? `UndecidableInstances` i was asked before to only enable explicitly in files that need it but these other two are probably fine